code/modules/antagonists/changeling/changeling.dm

```diff
@@ -516,6 +516,7 @@
 	user.socks = chosen_prof.socks
 
 	chosen_dna.transfer_identity(user, 1)
+	user.Digitigrade_Leg_Swap(!(DIGITIGRADE in chosen_dna.species.species_traits))
```

Comment:
Check that the species actually has `DIGITIGRADE` before calling this proc, as per the other two calls of this.